tests/Functional/Controller/Post/PostFrontControllerTest.php

```diff
@@ -216,6 +214,9 @@ private function prepareEntries(): KernelBrowser
             $this->getUserByUsername('JaneDoe')
         );
 
+        // sleep so the creation time is actually 1 second apart for the sort to reliably be the same
+        sleep(1);
```

Comment:
Introducing sleep is never a good idea, also not in tests. Depending on the CPU usage, performance, resources etc. This could change..

If you want to make it more robust, look into a polling technique or maybe a notify system. But `sleep()` is actually always guaranteed to be a problem sooner or later.

Reply:
Ah I got it. This might be robust enough.